packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -284,12 +295,37 @@ export class TaggedLoggerAdapter implements ITelemetryBaseLogger {
 	}
 }
 
+const childLoggerType = "__fluid_child_logger" as const;
```

Comment:
I wonder if we have convention on the formatting of this kind of tag.  Might be nice